### PR TITLE
Add masseffect.com and ridygo.fr. Remove customelements.io

### PR DESCRIPTION
**Pull request by Link2Twenty**
Removed: 
 - Customelements.io

Added:
 - Masseffect.com
 - Ridygo.fr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,6 @@ Sorted alphabetically
 | [:link:](https://www.cerberusapp.com) | Cerberus | `1.8` | | |
 | [:link:](https://www.cloudstitch.com) | Cloudstitch | `1.8` | | |
 | [:link:](https://ux.contactlab.com) | ContactLab - UI components | `1.8` | | [Source](https://github.com/contactlab/contactlab-ui-components) |
-| [:link:](https://customelements.io) | CustomElements | `1.1` | | [Source](https://github.com/customelements/www) |
```
Comment (abdonrd) [inline README.md:58]: What do you think about strike the project name?

Comment (Link2Twenty) [reply README.md:58]: That would work, or a new table of projects no longer around with a web archive link?

Comment (abdonrd) [reply README.md:58]: That last sounds good!

**Comment by Link2Twenty** [comment]
It seems web archive is a no go (it wasn't happy to do the importing of elements)
So I've added customelement.io back in with a strikethrough

**Comment by abdonrd** [comment]
Thanks @Link2Twenty!

**Pull request merged**